docs/luigi-client-api.md

```diff
@@ -109,23 +153,42 @@ Removes a custom message listener.
 **Meta**
 
 -   **since**: 0.6.2
+    LuigiClient.removeCustomMessageListener(customMsgId)
 
 #### getToken
 
 Returns the currently valid access token.
 
+##### Examples
+
+```javascript
+const accessToken = LuigiClient.getToken()
+```
+
 Returns **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** current access token
 
 #### getEventData
 
 Returns the context object. Typically it is not required as the [addContextUpdateListener()](#addContextUpdateListener) receives the same values.
 
+##### Examples
+
+```javascript
+const context = LuigiClient.getEventData()
+```
+
 Returns **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** current context data
 
-#### getContext
+#### getEventData
 
 Returns the context object. It is an alias function for getEventData().
 
+##### Examples
+
+```javascript
+const context = LuigiClient.getContext()
+```
+
```

Comment:
The title changed to `getEventData` but the example code is still using `getContext()` . 

In the screenshot getEventData seems to have ended up a duplicate 🤔 :  
![Screenshot 2020-08-17 at 12 17 44](https://user-images.githubusercontent.com/18335489/90385761-d78e8280-e083-11ea-853a-a63540f93d07.png)

Reply:
@maxmarkus @ndricimrr I see getEventData() is already described above so getContext should stay as it was, no need to remove it and the example as well. What do you think, guys?

Reply:
The example codes seem to be in place. The second title I guess was intended for `getContext` instead of `getEventData`. When searching for `getContext` it should then go to the second title in the screenshot

Reply:
`getEventData` was first and after confusions we introduced `getContext` which does the same. But we go with `getEventData` as well because of backwards compatibility.

Reply:
I have used `* @alias getEventData` and this seems to break it. Guess we should set `getEventData()` as deprecated, to not confuse people with two equal functions.

Reply:
I have moved up `getContext` and removed the example for `getEventData`, which is set now as deprecated.
<img width="942" alt="Proposal" src="https://user-images.githubusercontent.com/1720843/91038794-ac6bdc00-e60b-11ea-8e6b-940de692b39c.png">